RELEASE_NOTES.md

```diff
@@ -2,32 +2,16 @@
 
 ## Summary
 
+<!-- Here goes a general summary of what this release is about -->
+
 ## Upgrading
```

Comment:
Can you remove the Upgrading section?  Now that we have the "Bug fixes" section, we don't need this anymore.

Reply:
And should probably be renamed to `Breaking Changes`, not deleted.